src/main/java/org/jabref/gui/util/ThemeLoader.java

```diff
@@ -38,6 +39,7 @@
 
     public static final String MAIN_CSS = "Base.css";
     public static final String DARK_CSS = "Dark.css";
+    private static String CUSTOM_CSS = "";
```

Comment:
This field is not really required for this class. I would prefer if only the preferences know about the current custom theme (in form of the `FX_Theme` field).